app/script/components/ephemeralTimer.js

```diff
@@ -29,16 +29,21 @@ z.components.EphemeralTimer = class EphemeralTimer {
     const dashLength = 12.6;
     const dial = componentInfo.element.querySelector('.ephemeral-timer__dial');
 
+    const numberOfAnimationSteps = 40;
```

Comment:
Did you test this value and check with design?

Reply:
I sad next to Thomas and we did a empirical approach with directly checking it on different values. We think it looks very smooth.